modules/solid_mechanics/src/neml2/models/LAROMANCE6DInterpolation.C

```diff
@@ -350,9 +364,8 @@ LAROMANCE6DInterpolation::json_vector_to_torch(const std::string & key) const
   if (!_json.contains(key))
     throw NEMLException("The key '" + std::string(key) + "' is missing from the JSON data file.");
 
-  std::vector<Real> in_data = _json[key].template get<std::vector<Real>>();
-  const int64_t data_dim = in_data.size();
-  return Scalar(torch::from_blob(in_data.data(), {data_dim}).clone());
+  std::vector<Real> in_data = _json[key].get<std::vector<Real>>();
+  return Scalar::create(in_data).clone();
```

Comment:
I didn't know about the libtorch create function, I didn't know there was another torch function for converting an array.  Thanks for fixing all of these.

Reply:
Yep @dschwen is correct.